### ISSUE TITLE
Bump up version of pip to 21.1.1

### ISSUE THREAD

**Issue by juanitosvq** [pypa/virtualenv#2102]
Hi there,

Would it be possible to bump up the version of `pip` running in `virtualenv`? All the versions less than `v21.1` are affected by a security vulnerability:
https://github.com/pypa/pip/pull/9827

The safety report:

```
+============================+===========+==========================+==========+
| package                    | installed | affected                 | ID       |
+============================+===========+==========================+==========+
| pip                        | 21.0.1    | <21.1                    | 40291    |
+==============================================================================+
| Pip 21.1 stops splitting on unicode separators in git references, which      |
| could be maliciously used to install a different revision on the repository. |
| See: <https://github.com/pypa/pip/issues/9827>. Additionally, pip 21.1       |
| updates urllib3 to 1.26.4 to fix CVE-2021-28363.               
```

Thanks in advance!

**Comment by MglMX** [comment]
You can run `virtualenv --upgrade-embed-wheels` to upgrade the pip version used by `virtualenv` but indeed it would be better to have a non vulnerable version by default.

**Comment by juanitosvq** [comment]
Fixed by https://github.com/pypa/virtualenv/pull/2104 available in `v20.4.5`. 
Thanks @gaborbernat !
